realsense_ros/src/rs_base.cpp

```diff
@@ -457,4 +457,8 @@ Result RealSenseBase::changeFPS(const stream_index_pair & stream, const rclcpp::
   }
   return result;
 }
+
+rclcpp::Time RealSenseBase::frameToTime(const rs2::frame & frame) {
+  return rclcpp::Time(frame.get_timestamp() * 1000);
```

Comment:
I believe `rclcpp::Time` takes nanoseconds as an input. As `get_timestamp` returns milliseconds, the result of this conversion is microseconds and not nanoseconds.

A more readable and consistent way would be to leverage the macros provided by rcl. These are the ones used by the core of ROS 2 for conversions. In this case `RCL_MS_TO_NS`
https://github.com/ros2/rcl/blob/ceff88b974303c51cba589f9992dd5fcc253ca57/rcl/include/rcl/time.h#L32
https://github.com/ros2/rcutils/blob/e7eb74c7a11cf207aa9330d77f44098339528cac/include/rcutils/time.h#L32

Reply:
Thanks for tip!